test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -318,43 +333,22 @@ def _undefined
   end
 
   # Set up grpc stubs to mock the external calls.
-  # TODO(qingling128): Remove this comment after grpc/grpc#12506 is resolved.
-  # Due to a gRPC load balancing issue (grpc/grpc#12506), we have to use a
-  # different port each time we create a gRPC mock as a temporary workaround.
-  # Thus we can only create one driver in each setup_logging_stubs context.
-  def setup_logging_stubs(should_fail = false, code = 0, message = 'Ok')
-    # Save the mock host in an instance variable, so later on when creating the
-    # logging driver, we can refer to this host with exactly the same port.
-    @mock_host = generate_mock_host
-    srv = GRPC::RpcServer.new
+  def setup_logging_stubs
```

Comment:
Is this now effectively a no-op? Where are the actual stubs set up? Shouldn't that be here?